agents-manage-ui/src/app/[tenantId]/projects/page.tsx

```diff
@@ -45,8 +45,8 @@ async function ProjectsPage({ params }: PageProps<'/[tenantId]/projects'>) {
             description={emptyStateProjectDescription}
             action={
               <NewProjectDialog tenantId={tenantId}>
-                <Button size="lg" className="gap-2">
-                  <Plus className="h-4 w-4" />
+                <Button size="lg">
+                  <Plus />
```

Comment:
<img width="1078" height="580" alt="image" src="https://github.com/user-attachments/assets/d6770d66-b21c-49b2-ad33-77500e413ce3" />
before it was `Create project`